src/databases/Claw/avtClawFileFormat.C

```diff
@@ -1263,7 +1266,7 @@ avtClawFileFormat::GetVar(int timeState, int domain, const char *varname)
         EXCEPTION1(InvalidFilesException, oss.str().c_str());
     }
     lseek(fd, dsOffset, SEEK_SET);
-    size_t nread =
+    int nread =
```

Comment:
In my read of `man 2 read`, the return value should be an `ssize_t`

Reply:
you are correct, I was mislead by some google searches. 

To do error checking property we need to check for 0 and then look at the error codes.  I'll try to tackle that. 

(Note: we also have another instance of using int as the return value for this call, we only had one place that was using size_t)

[ ] cyrus fix this

Reply:
The variant we use on windows *does* have int return value.

This is turning into wack-a-mole. 

@biagas any idea how we should attack this? Do we have a VisIt general wrapper for read?

Reply:
I'm not aware of a wrapper off-hand.  Perhaps modify like so:

```c++
#ifndef WIN32
        ssize_t nread = read(fd, buf, sizeof(buf)-1);
        if (nread >= sizeof(buf)-1)
#else
        int nread = ::_read(fd, buf, sizeof(buf)-1);
        if (nread < 0)
            // error with read, we should do something
        else if (nread >= (int)sizeof(buf)-1)
#endif
```

Reply:
This can't be the only instance in our database plugins where we're looking at the return value of a `read()` system call.

Reply:
There are a few more cases, the warning just tipped us off to this particular one.

I am going to file another ticket for a broader audit of our use of read() vs ::_read()

Are we ok with using the `int` in this PR? It matches the rest of the logic (several other read calls) in the Claw plugin/

Reply:
Sure, no problem